corehq/apps/api/urls.py

```diff
@@ -172,15 +172,17 @@ def api_url_patterns():
 USER_API_LIST = GLOBAL_USER_API_LIST + NON_GLOBAL_USER_API_LIST
 
 
-def get_global_api_url_patterns(resources):
+def _get_global_api_url_patterns(resources):
```

Comment:
Nit: might want to call this `_get_global_api_url_pattern` now since it returns one thing rather than generating a few things.

Reply:
Though it's basically just a single top node with things under it (the `include`) so maybe you could interpret it as correct either way. Fine to leave